crates/aptos/src/update/revela.rs

```diff
@@ -131,6 +135,16 @@ impl CliCommand<String> for RevelaUpdateTool {
     }
 
     async fn execute(self) -> CliTypedResult<String> {
+        if self.check {
```

Comment:
Can you find a way to dedupe this code?

Reply:
I don't think this is going to be particularly pretty to dedupe, without a bunch of extra abstraction :/